tests/integration-tests/tests/schedulers/test_slurm_accounting/resources/run_jobs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+_node_count=$([ -n "$1" ] && echo "$1" || echo 4)
+
+cat > runme.sh << EOF
+#!/bin/bash
+MY_NAME=\$(hostname)
+echo "\${MY_NAME} \${SLURM_JOB_ACCOUNT} \${SLURM_JOB_ID} \${SLURM_JOB_NAME}"
+EOF
+chmod +x runme.sh
+srun -N "$_node_count" runme.sh
```

Comment:
As reported in a comment above, I would dismiss this file and use the `slurm_commands` class and its methods to run test jobs on the cluster.